dowhy/gcm/anomaly_scorers.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional
+
+import numpy as np
+from statsmodels.robust import mad
+
+from dowhy.gcm.constant import EPS
+from dowhy.gcm.anomaly_scorer import AnomalyScorer
+from dowhy.gcm.density_estimator import DensityEstimator
+from dowhy.gcm.util.general import shape_into_2d
+
+
+class MedianCDFQuantileScorer(AnomalyScorer):
```

Comment:
will be good to add a docstring for this class. What does the score imply statistically? an equation/formula will help.

Reply:
Added a docstring.